prime-router/metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform.yml

```diff
@@ -276,3 +277,48 @@ elements:
     resource: '%diagnosticReport.basedOn.resolve().requester.resolve().organization.resolve().telecom.where(system = "phone")'
     resourceIndex: telecomIndex
     schema: datatypes/simple-report-telecom
+
+  # populate Specimen identifier
```

Comment:
Maybe a little more comment around why these are required?  It looks like the schema was previously falling back to the the performer on the diagnostic report.

It would be good to capture that here so we can work with SR to update it